### ISSUE TITLE
OMEMO in a MUC

### ISSUE THREAD

**Issue by genofire** [conversejs/converse.js#1180]
It shows `Unencryptable OMEMO message`

**Comment by genofire** [comment]
![screenshot from 2018-09-08 13-16-16](https://user-images.githubusercontent.com/6905586/45253553-c6498100-b358-11e8-8103-eb8d98cceeac.png)

**Comment by allo-** [comment]
Same here. Neither incoming messages nor carbons are decrypted. Outgoing messages are encrypted with all relevant keys.

**Comment by genofire** [comment]
Work carbons in singlechat? (Have not try to devices at the same time)

**Comment by genofire** [comment]
i check it also again - reading OMEMO works :+1: 
now only writing is nessesary

**Comment by trasba** [comment]
> Work carbons in singlechat? (Have not try to devices at the same time) - that would be another issue

Hi,

I have exactly this issue. Setup is prosody server with xep-0280 'carbons'
It is working for unencrypted messages. OMEMO encrypted messages can be received and decrypted but I can't send any encrypted messages. I get:

> Sorry, could not send the message due to an error. Cannot read property 'prekeys' of undefined

Setup is both clients are conversations, server is prosody and one of the clients is logged in with conversejs in parallel.

EDIT: Curiously it is now working I just deactivated carbons now on the server and activated it again. Then I got the error message constantly I logged in from the conversejs website and than it worked. It is now also working from my conversejs website. Very strange but ok...
So just to let you know it is working
